### PR TITLE
Removed <div> wrapper from <button>

### PR DESCRIPTION
**Pull request by muskankhedia**
_Fixes: #413_

Removed the div wrapper from the button tags in the files.

### CODE REVIEW

**Comment by muskankhedia** [comment]
@vdvibhu20 Pls review the PR. The changes are made according to the latest code available up to the date.

**Comment by vdvibhu20** [comment]
file `app/pods/components/code-editor/template.hbs:` still has div wrapper and many other files

**Comment by muskankhedia** [comment]
I have made the changes, pls do have a look.

**Comment by vdvibhu20** [comment]
remove conflicts and sync your branch with `coding-blocks:master`.

**Comment by muskankhedia** [comment]
@vdvibhu20 pls review the PR, I have solved the merge conflicts.

File: app/pods/components/certificate-comp/template.hbs [hunk 1]
```diff
@@ -116,7 +116,15 @@
             <button>Generate Now!</button>
           </div>
         {{/if}}
+<<<<<<< HEAD
```
Comment (championswimmer) [inline app/pods/components/certificate-comp/template.hbs]: The merge conflict demarcaters are still here

**Comment by muskankhedia** [comment]
@championswimmer Pls review this.

**Pull request closed without merging**